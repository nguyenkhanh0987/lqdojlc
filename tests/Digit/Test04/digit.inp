100
37508 56237
12205 31423
46206 98381
50837 72013
74431 88035
88515 99547
69863 91751
57564 58543
86327 94698
58978 89138
92949 93105
84513 88277
35755 93955
55923 72979
41244 77658
76503 93961
3536 29475
32181 73823
35040 69615
6818 37900
40247 60879
65051 82596
37987 71229
65547 74726
57392 95015
50984 82150
25756 35380
93885 98036
48652 49531
56937 68927
93804 96445
46127 53281
34320 91474
10131 87560
76241 81828
75814 99033
39631 64252
42063 51400
53644 80607
6550 64618
73919 78290
71803 77764
29492 65010
23405 77019
74504 97136
88269 96686
91107 99392
20865 95922
39935 88178
73115 80285
86519 95766
95041 99048
71227 97682
41525 84957
71647 90789
81497 89091
95170 99455
22257 62213
60340 81750
23213 49845
62948 94119
43466 72958
15377 22746
10759 78540
54059 55447
5734 89037
12842 30087
43168 74744
61058 95854
36749 41704
7819 54869
65039 72582
12155 95590
2223 58079
61785 72390
4101 69670
83260 89351
39336 93220
1916 9874
70827 84628
34252 68533
73096 92468
20525 46214
81525 87972
53156 92840
66243 80112
60468 71800
96245 97147
2171 51438
45157 92738
8045 11183
31117 99297
16485 87487
48257 74823
71265 72169
64767 67441
76565 90537
63273 84177
34974 82428
2895 45549
