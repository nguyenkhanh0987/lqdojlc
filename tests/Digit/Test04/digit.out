2003
3483
3310
1568
614
361
874
162
282
1681
3
183
4492
1342
2908
711
5242
4019
3721
5477
2008
1088
3229
582
1998
2072
1480
98
209
1037
53
829
4665
8345
370
931
2167
1103
1980
8184
283
243
3624
5640
881
209
230
6604
3413
483
291
155
963
3190
861
271
210
4944
1184
3919
1528
2572
1434
7981
125
9748
3363
2646
1575
796
6994
594
8414
8629
683
9351
308
3722
2003
535
3608
845
3415
163
2369
981
683
24
8142
3296
754
5130
7405
2002
5
231
706
1160
4266
7223
